### PR TITLE
Use gopath when calling lint and validate tests

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: ci-operator/config/containers/libpod/master.yaml [hunk 1]
```diff
@@ -17,13 +17,13 @@ build_root:
 tests:
 - artifact_dir: /tmp/artifacts
   as: validate
-  commands: ARTIFACT_DIR=/tmp/artifacts make validate
+  commands: ARTIFACT_DIR=/tmp/artifacts GOPATH=/go make validate
```
Comment (smarterclayton) [inline ci-operator/config/containers/libpod/master.yaml]: The build image already sets GOPATH, if you aren’t seeing it that means someone regressed the image.  @stevekuznetsov I might suspect the build root changes.

Comment (baude) [reply ci-operator/config/containers/libpod/master.yaml]: This is based on a Fedora image and not the openshift centos one.  Are you setting an image ENV for GOPATH? If so, I could do that on the Fedora image too.

Comment (stevekuznetsov) [reply ci-operator/config/containers/libpod/master.yaml]: Wait, `clonerefs` upsteam will set up GOPATH to be correct -- if the clone for `ci-operator` doesn't own this, it needs to. It should be invisible to the user.

Comment (baude) [reply ci-operator/config/containers/libpod/master.yaml]: I don't think it exports it though?

Comment (stevekuznetsov) [reply ci-operator/config/containers/libpod/master.yaml]: Yeah that is a bug on our end :(

**Pull request closed without merging**